modules/displayPokemon.js

```diff
@@ -0,0 +1,78 @@
+import { getComments } from './involvementRequests.js';
+import displayMessage from './displayMessage.js';
+/* eslint-disable no-new */
+const cardPoke = document.querySelector('.cardModal');
+const $grafica = document.querySelector('#grafica');
+
+const displayPokemon = async (card, index) => {
+  let nroAbility = 0;
+  let datosIngresos = {};
+  const etiquetas = [];
+  const datas = [];
+  // eslint-disable-next-line no-undef
+  let pokeinfo = '';
+  pokeinfo += `
+    
+    <img src="${card[index].data.sprites.other['official-artwork'].front_default}" alt="pokemon">
+    <div id="info">
+    <h2>${card[index].data.name.toUpperCase()}</h2>
+    `;
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
Please don't disable the linters. Consider fixing the linter issues instead of disabling the linters.